### ISSUE TITLE
DWMBlurGlass is not compatible with Open-Shell

### ISSUE THREAD

**Issue by watchofficial** [Maplespe/DWMBlurGlass#365]
**Describe the bug**
When DWMBlurGlass is running, launch Open-Shell menu would lead to DWM crash immediately (Windows Explorer also restart).
It seems like I cannot use both programs at the same time.

**To Reproduce**
1. Install/Run Open-Shell and setup with any settings.
2. Run DWMBlurGlass and click install with any settings.
3. Open Open-Shell start menu and hover the mouse on any items.

**Screenshots**
![15072024](https://github.com/user-attachments/assets/c7022c6b-4fba-4ad5-8095-d50c5c7d8d42)

**Additional information:**
OS Version : Windows 10 22H2 Build 19045.4529
Software Version : 2.2.0

**What other third-party software do you use:**
Open-Shell v4.4.191
7+ Taskbar Tweaker v5.15.1 (I think this is not related)

**Additional context**

**Comment by T0biasCZe** [comment]
I use Open Shell v4.4.191 with DwmBlurGlass 2.2.0 and it works for me, so it may be issue on your PC...
Unrelated, but what Open Shell skin is that? 😅

**Comment by watchofficial** [comment]
>Unrelated, but what Open Shell skin is that? 😅

Windows 7 Like Combo revE (on classic shell website iirc)

**Comment by SJL64** [comment]
I have tried with the latest stable version of open shell plus a few other older versions down to classic shell and I always have flickering problems with glass over multiple rows menus mostly will change to a shade that is close to opacity and colour variant I am using for the skin. 

Both win 11+12 I am wondering if the people not having issues are using full glass or a dark glass skin in open shell...sometimes will need to open dwm app and uninstall reinstall sometimes not and it's ok. 

I would love to get to the bottom of this as it's one of few things holding my system back now, I'm going pretty easy on the bloat ware these days but this is an essential app for me. 

Anybody tinkering with some integration of these two popular tweaker apps? 

Yes we have reinstalled the graphics driver plenty of times 😞
